### PR TITLE
Line edit

### PR DESCRIPTION
**Pull request by richa208**
addresses #96 

## What is included in this PR?
changes are made to : 
- briefcase-api.rst
- briefcase-forms.rst

### CODE REVIEW

File: briefcase-forms.rst [hunk 1]
```diff
@@ -10,19 +10,19 @@ Pulling forms from ODK Aggregate
...
-- In the window which opens up select :guilabel:`pull` tab.
-- Select Aggregate 0.9.x or Aggregate 1.0 in the :guilabel:`pull data from` the drop-down.
-- Enter the url of your ODK Aggregate server in :guilabel:`url` option. For instance `https://proj1.appspot.com` and click on connect.
-- You can either leave the username blank if the anonymousUser is granted Data Viewer permissions or specify an ODK Aggregate user (Account Type 'ODK') that has been granted Data Viewer permissions.
-- A list of forms will be displayed. Select the ones you want to pull and click on :guilabel:`pull` button at the bottom of the window. The forms will be pulled to ODK Briefcase Storage on your local system.
+- In the window which opens up select: guilabel: pull` tab.
+- Select Aggregate 0.9.x or Aggregate 1.0 in the: guilabel: pull data from` the drop-down.
+- Enter the URL of your ODK Aggregate server in guilabel:`url` option. For instance `https://proj1.appspot.com` and click on connect.
+- You can either leave the username blank if the anonymous user is granted Data Viewer permissions or specify an ODK Aggregate user (Account Type 'ODK') that has been granted Data Viewer permissions.
```
Comment (yanokwa) [inline briefcase-forms.rst]: The case here is important because it matches the Aggregate UI.

File: briefcase-forms.rst [hunk 2]
```diff
@@ -10,19 +10,19 @@ Pulling forms from ODK Aggregate
...
+- A list of forms will be displayed. Select the ones you want to pull and click on: guilabel:`pull` button at the bottom of the window. The forms will be pulled to ODK Briefcase Storage on your local system.
 
 .. warning::
 
  ODK Briefcase will abort a pull attempt if the local copy of the form definition file differs in any way from the copy being pulled from the server or ODK Collect device.
 
 .. tip::
 
- If the form definition has changed but the changes only affect the question text and do not alter the structure of the collected data (or change the formId or version), you can create a new ODK Briefcase Storage area, pull data into that, then manually copy the instances from there into your original storage area.
+ If the form definition has changed but the changes only affect the question text and do not alter the structure of the collected data (or change the formed or version), you can create a new ODK Briefcase Storage area, pull data into that, then manually copy the instances from there into your original storage area.
```
Comment (yanokwa) [inline briefcase-forms.rst]: This should be reverted. It's `formId` as in form identifier.

File: briefcase-forms.rst [hunk 3]
```diff
@@ -109,45 +109,45 @@ Export forms to CSV
...
+In Briefcase v1.4.4 and later, there is a scriptable command-line interface.
 
 Pulling form data from Aggregate Server
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .. code-block:: console
 
   $ java -jar "ODK Briefcase v1.4.4 Production.jar" --form_id market_prices --storage_directory ~/Desktop --aggregate_url https://my_server.appspot.com --odk_username my_username --odk_password my_password
 
-This command pulls form data with id market_prices from Aggregate server at `https://my_server.appspot.com` and stores data in Briefcase's storage directory on the :file:`~/Desktop`
+This command pulls from data with id market_prices from Aggregate server at `https://my_server.appspot.com` and stores data in Briefcase's storage directory on the: file:`~/Desktop`
```
Comment (yanokwa) [inline briefcase-forms.rst]: This should be reverted.

File: briefcase-api.rst [hunk 4]
```diff
@@ -21,7 +21,7 @@ The APIs used:
 
 1. OpenRosa Form Discovery API
 2. download forms and media files using URLs form discovery.
-3. ``view/submissionList`` to obtain a chunk of submission keys.
+3. ``view/submissionlist`` to obtain a chunk of submission keys.
```
Comment (yanokwa) [inline briefcase-api.rst]: I'm not 100% sure that the API requires it, but to be safe, it should be case-sensitive. This change should be reverted.

File: briefcase-api.rst [hunk 5]
```diff
@@ -31,9 +31,9 @@ Push
 
 Interfaces used during Push actions.
 
-1. *formUpload* to upload a form and its media files to ODK Aggregate.
+1. *form upload* to upload a form and its media files to ODK Aggregate.
```
Comment (yanokwa) [inline briefcase-api.rst]: I'm not 100% sure that the API requires it, but to be safe, it should be case-sensitive. This change should be reverted.

**Comment by richa208** [comment]
@yanokwa Sorry for the trouble :sweat_smile: I have made the requested changes!

**Comment by yanokwa** [comment]
Thanks for sending in this PR, @richa208. I'm looking forward to your next!

**Comment by richa208** [comment]
Sure @yanokwa :smiley:

**Pull request merged**